cedar-policy-validator/src/cedar_schema/to_json_schema.rs

```diff
@@ -237,7 +228,8 @@ fn convert_qual_name(qn: Node<QualName>) -> json_schema::ActionEntityUID<RawName
 
 // Convert the applies to decls
 fn convert_app_decls(
-    action_info: (&SmolStr, &Loc),
+    name: &SmolStr,
+    loc: &Loc,
```

Comment:
maybe a comment describing what this is the `Loc` of -- just the `name`? the whole block? something in between?